runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -589,7 +589,15 @@ else if (!cid.getItem().hideValue().equals(""))
 
 			if (cid.getType() == int.class)
 			{
-				int value = Integer.parseInt(configManager.getConfiguration(cd.getGroup().value(), cid.getItem().keyName()));
+				int value;
+				try
+				{
+					value = Integer.parseInt(configManager.getConfiguration(cd.getGroup().value(), cid.getItem().keyName()));
+				}
+				catch (NumberFormatException ex)
+				{
+					return;
```

Comment:
is there a way to fallback to the default config value here?

Reply:
That happens when a plugin tries to get the value of the config option

For example change ```gpu.drawDistance``` in your config to ```gpu.drawDistance=test```

If you start the client it will throw an NumberFormatException, when you login the value is reset to the default value (25).

Reply:
lgtm then